EMMO imports and redefinition

hothello opened the pull request:
The ontology currently imports the whole EMMO. In this commit, `owl:imports` point only to the EMMO modules effectively used.
An axiom is added to the classes Solvent and Solute that states:
`isPartOf some Solution`.
A similar axiom is added to `emmo:AqueousSolution` stating:
`hasPart some water`

jsimonclark commented:
Thank you for your pull request! Just two questions:

- We originally chose to import the entire EMMO instead of specific modules because the speed is comparable and we took the view that it is better to have too much than too little. Is there a good reason to limit the imports, in your view?
- The Solution axioms are fine. We have dedicated `hasSolute `and `hasSolvent `object properties. Would you rather use those instead of `hasPart `and `isPartOf `?

hothello commented:
>     We originally chose to import the entire EMMO instead of specific modules because the speed is comparable and we took the view that it is better to have too much than too little. Is there a good reason to limit the imports, in your view?

Importing the whole EMMO is slower because Protégé has to download more files remotely[^1]. Also, having too many leaves is considered a bad practice (see [Towards Ontology Quality Assessment](https://ceur-ws.org/Vol-1824/ldq_paper_2.pdf) ). Finally, using only the necessary EMMO modules results in a smaller inferred ontology.

>     The Solution axioms are fine. We have dedicated `hasSolute` and `hasSolvent` object properties. Would you rather use those instead of `hasPart` and `isPartOf`?

Thanks for pointing this out, I overlooked these properties. I have made the following changes in the last commit:
* Moved the properties `hasSolute` and `hasSolvent` as subproperties of `emmo:hasComponent` and their range expanded to include `emmo:Component`.
* A new property `:isComponentOf` has been added and used to connect `Solute` and `Solvent` to `emmo:Solution`. In this way, the inferred ontology correctly classifies them as being part of an `emmo:HolisticSystem`, e.g. one where the parts are indistinguishable.
* Added the axioms `hasSolute some Solution` and `hasSolvent some Solvent` to `emmo:Solution`, which is now inferred to be an `emmo:HolisticSystem`. This is, in my opinion, very consistent with the EMMO philosophy.
* `hasSolvent some Water` added to `emmo:AqueousSolution`.


[^1]: Unnecessary internet traffic comes with its carbon footprint too, although it is undoubtedly much lower than other online services.

jsimonclark commented:
I discussed the point about importing all of EMMO or just modules with @gerhardgoldbeck and others. We're converging on the idea that ontologies under active development should import all of EMMO and strip down to only the necessary modules with the stable release. This is because it's not always clear which modules you need when doing development and we want to avoid people inadvertently creating duplicate terms. So, I will keep the full import for now and make it a point to strip it down when we do the 1.0.0 release. 

The solution axioms are fine.
